src/modules/update/Kconfig

```diff
@@ -0,0 +1,12 @@
+menuconfig USE_AUTO_UPDATE
+    bool "Auto Update Options"
+    default y
+    help
+        Allow Soletta built apps to be updated via HTTP connection.
```

Comment:
do not mention HTTP. That's backend-specific.

Say a small OS device may use OIC or CoAP or simply UDP to get the packets... that are downloaded straight to EEPROM.